.circleci/config.yml

```diff
@@ -30,14 +30,16 @@ jobs:
       - checkout
 
       - restore_cache:
-          key: syndesis-rest-m2
+          key: syndesis-rest-m2-{{ checksum "pom.xml" }}-{{ checksum "connector-catalog/pom.xml" }}-{{ checksum "controllers/pom.xml" }}-{{ checksum "core/pom.xml" }}-{{ checksum "credential/pom.xml" }}-{{ checksum "dao/pom.xml" }}-{{ checksum "github/pom.xml" }}-{{ checksum "jsondb/pom.xml" }}-{{ checksum "model/pom.xml" }}-{{ checksum "model2/pom.xml" }}-{{ checksum "openshift/pom.xml" }}-{{ checksum "project-generator/pom.xml" }}-{{ checksum "rest/pom.xml" }}-{{ checksum "syndesis-maven-plugin/pom.xml" }}-{{ checksum "runtime/pom.xml" }}-{{ checksum "verifier/pom.xml" }}-{{ checksum "inspector/pom.xml" }}-{{ checksum "setup/pom.xml" }}
+          key: syndesis-rest-m2-{{ checksum "pom.xml" }}
```

Comment:
Double key ?

Reply:
Yes that's a fallback.

Reply:
I think we should add a `syndesis-rest-m2` as a final fallback key just for fun.